funcx_web_service/routes/funcx.py

```diff
@@ -212,21 +212,33 @@ def submit(user: User):
         # this should raise a 500 because it prevented any tasks from launching
         return create_error_response(RequestKeyError(e), jsonify_response=True)
 
+    batch_id = str(uuid.uuid4())
```

Comment:
We could try to parse the `post_req` for a `taskgroup_id` field and if it's missing mint an id. I'm not sure if we want a situation where we do not create a batch at all.